modules/web/src/main/scala/observe/ui/Icons.scala

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package observe.ui
+
+import react.fa.FAIcon
+import react.fa.FontAwesomeIcon
+import react.fa.IconLibrary
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.*
+
+object Icons:
```

Comment:
I wonder is some of these could go to `lucuma-ui` Maybe it just make things complicated?

Reply:
IDK what's best here, whether to have everything in one place or each system with their own references for the icons they use.